app/router.js

```diff
@@ -35,6 +35,14 @@ Router.map(function() { // eslint-disable-line
     this.route('archive');
     this.route('upcoming');
     this.route('reviewed');
+
+    this.route('project', {
+      path: ':project_id',
+    }, function() {
```

Comment:
sorry, tiny tiny nit, but this nested route syntax should look like the other nesting going on

Reply:
gotcha